EIPS/eip-3675.md

```diff
@@ -0,0 +1,246 @@
+---
+eip: 3675
+title: Upgrade consensus to Proof-of-Stake
+author: Mikhail Kalinin (@mkalinin), Danny Ryan (@djrtwo), Vitalik Buterin (@vbuterin)
+discussions-to: https://ethereum-magicians.org/t/eip-3675-upgrade-consensus-to-proof-of-stake/6706
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-07-22
+requires: 2982
```

Comment:
Meta question 1: can a core EIP depend on an informational EIP?

Meta question 2: should 2982 be finalised first?

Reply:
The answer to the second is that it this just can't surpass the status of its dependency.  So if the dependency is in Last Call, this can only go up to Last Call.

For the first, I'm not a fan of a Core EIP depending on an Informational EIP, but I have largely lost that debate at this point I think.

Reply:
I don't have a strong opinion either way.

Reply:
I am happy to remove this dependency as the specification part of this EIP does not depend on EIP-2982. References to EIP-2982 throughout the doc are purely informational.

Reply:
yeah, fine by me